web-registry/src/ledger.tsx

```diff
@@ -59,10 +64,11 @@ export const LedgerProvider: React.FC = ({ children }) => {
   const [api, setApi] = useState<RegenApi | undefined>(undefined);
   const [loading, setLoading] = useState<boolean>(false);
   const [error, setError] = useState<unknown>(undefined);
+  const { wallet } = useWallet();
```

Comment:
using `useWallet` here to get the offline signer and provide it to the regen api
cc/ @victordelval @mhagel